src/main/java/com/example/controller/BookController.java

```diff
@@ -35,4 +37,10 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.createBook(bookDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @PutMapping("/{id}")
```

Comment:
I guess you misunderstood my previous comment. You need to provide separate endpoints for updating and deleting the book. Following task description:
```
PUT: /api/books/{id} (Update a specific book)
DELETE /api/books/{id} (Delete a specific book)
```